### PR TITLE
APM known issues: Too many small bulk requests

### PR DESCRIPTION
**Pull request by carsonip**
Add APM known issues about too many small bulk requests.

Related bugfix: https://github.com/elastic/apm-server/pull/13576

### CODE REVIEW

File: docs/en/observability/apm/known-issues.asciidoc
Comment (inge4pres) [inline docs/en/observability/apm/known-issues.asciidoc:290]: Do we keep this open to the possibility of having the fix added to 8.14.3?

Comment (carsonip) [reply docs/en/observability/apm/known-issues.asciidoc:290]: Current info points at the earliest availability at 8.15.0. We can always update the docs if anything changes.

Comment (carsonip) [reply docs/en/observability/apm/known-issues.asciidoc:290]: I'll leave it to @bmorelli25 to suggest a more efficient workflow. Currently I'll update the docs when the bugfix is actually released, but there may be a way to word it so that we don't need to change "A fix will be released..." to "A fix was released..." later.

**Comment by carsonip** [comment]
@bmorelli25 can you check if the backports to 8.13 and 8.14 are appropriate? The known issue affects 8.13 and 8.14.

File: docs/en/observability/apm/known-issues.asciidoc
Comment (carsonip) [inline docs/en/observability/apm/known-issues.asciidoc:284]: do not merge the PR yet: I'm testing this.

Comment (carsonip) [reply docs/en/observability/apm/known-issues.asciidoc]: Added a section specifically for Elastic Cloud.

**Review by inge4pres — APPROVED** [review]

**Review by bmorelli25 — APPROVED** [review]

**Pull request merged**
